Prevent booking overlap

jcbollinger opened the pull request:
Updates the overlap() method of the Booking model to prevent certain overlaps where one booking's range is entirely contained in the other's, and the two either start or end at exactly the same time (or both).

Fixes #118

jcbollinger committed:
Prevent booking overlap

Updates the overlap() method of the Booking model to prevent certain
overlaps where one booking's range is entirely contained in the
other's, and the two either start or end at exactly the same time
(or both).

Fixes #118

delarosatrevin commented:
Hi @jcbollinger, thanks for catching this issue and proposing a fix. I will have a look and merge the changes.

jcbollinger committed:
Fixied session list data content for unscheduled sessions.

Fixed a regression in the sessions_list data content function, wherein it
throws an exception if it encounters a session whose start timestamp is
not set while it is filtering out older sessions.  This fix causes sessions
with no start date to pass that filter.

Fixes #120

jcbollinger commented:
Sorry, I got the fix for #120 rolled into this PR after the fact.  That was not intentional.

delarosatrevin commented:
No problem, I can check all the changes together

jcbollinger committed:
Merge branch '3dem:devel' into devel
